CMakeLists.txt

```diff
@@ -190,12 +192,18 @@ if(HIP_FOUND AND Libva_FOUND)
       ROCJPEG_ROCP_REG_VERSION_PATCH=${VERSION_PATCH})
   endif()
 
+  #Generate BUILD_INFO
+  configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/api/rocjpeg_version.h.in ${CMAKE_CURRENT_BINARY_DIR}/rocjpeg_version.h @ONLY )
+  install(TARGETS ${PROJECT_NAME} EXPORT rocjpegTargets DESTINATION ${CMAKE_INSTALL_LIBDIR} COMPONENT dev)
```

Comment:
Other ROCm components utilize the naming convention `componentname-targets`. Therefore, please change `rocjpegTargets` to `rocjpeg-targets`.

Reply:
@AryanSalmanpour - name changed with the latest update